### PR TITLE
Align templates with prototypes from illumos-gate

### PR DESCRIPTION
**Pull request by mtelka**

### CODE REVIEW

File: transforms/manifest-metadata-template [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # 1.0 of the CDDL.
 #
 # A full copy of the text of the CDDL should have accompanied this
-# source. A copy of the CDDL is also available via the Internet at
+# source.  A copy of the CDDL is also available via the Internet at
```
Comment (pyhalov) [inline transforms/manifest-metadata-template:8]: Is there something wrong with spaces?

File: transforms/copyright-template [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 # This file and its contents are supplied under the terms of the
-# Common Development and Distribution License ("CDDL"). You may
-# only use this file in accordance with the terms of the CDDL.
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
 #
 # A full copy of the text of the CDDL should have accompanied this
-# source. A copy of the CDDL is also available via the Internet at
+# source.  A copy of the CDDL is also available via the Internet at
```
Comment (pyhalov) [inline transforms/copyright-template:8]: Why do we have 2 spaces here?

Comment (alarcher) [reply transforms/copyright-template:8]: FYI I removed the double space in our template copyrights when I bumped to 2017. I thought it was a typo on our end.

Comment (mtelka) [reply transforms/copyright-template:8]: https://en.wikipedia.org/wiki/Sentence_spacing#Digital_age
http://src.illumos.org/source/xref/illumos-gate/usr/src/prototypes/prototype.Makefile#8

Comment (alarcher) [reply transforms/copyright-template:8]: OK. Previously we had a mix-and-match of both (first sentence had a single space) so I modified what thought was a typo. I do not mind as long as it is consistent.

**Review by alarcher — APPROVED** [review]

**Pull request merged**